src/service/brokers/http/connection.toit

```diff
@@ -17,6 +17,7 @@ class HttpConnection_:
   root-certificates_/List? := null
 
   constructor .network_ .config_:
+    certificate-roots.install-common-trusted-roots
```

Comment:
Can we avoid the work for every single http connection? Why not just do this in `run-artemis` or when setting up the synchronization job?